README.md

```diff
@@ -235,6 +235,10 @@ Options passed to `start_link/2` as the options `{client_opts, [...]}`.
   cluster client and false otherwise. (Not useful if the client is used outside
   of a cluster.)
 
+* `{auth, {Username :: binary(), Password :: binary()}}`
```

Comment:
Shouldn't we also support `AUTH <password>` when `requirepass` is configured?

Reply:
From Redis 6 you can use "default" as username in this case. Isn't that enough?

Reply:
Ah, didn't know about `default`, maybe that can be added to the text.
Since we only have compability testing for Redis `6.2.11` and `7.0.9` in CI (which indicates supported version) it can be enough.